whisper-fine-tuning-sprint/README.md

```diff
@@ -0,0 +1,617 @@
+# Whisper fine-tuning sprint 🤗
+
+Welcome to the Whisper fine-tuning sprint 🎙️ !
+
+The goal of this event is to build **robust**, **real-world** speech recognition (ASR) systems in as many languages as possible 🌏🌍🌎.
+If necessary and available, free access to a V100s 32 GB GPU will kindly be provided by the [PUT CLOUD PROVIDER HERE](https://PUTCLOUDPROVIDERHERE.COM) 🚀.
```

Comment:
Fingers crossed 🤞

Reply:
To unblock this PR, maybe just say this  

```suggestion
Free compute resources will be provided for those that cannot train models with their compute resources.  🚀.
```
(or something along those lines)